packages-exp/auth-exp/src/model/public_types.ts

```diff
@@ -252,16 +252,16 @@ export interface Auth {
    */
   languageCode: string | null;
   /**
-   * The Auth instance's tenant ID.
+   * The {@link Auth} instance's tenant ID.
    *
    * @remarks
-   * This is a readable/writable property. When you set the tenant ID of an Auth instance, all
+   * This is a readable/writable property. When you set the tenant ID of an {@link Auth} instance, all
    * future sign-in/sign-up operations will pass this tenant ID and sign in or sign up users to
    * the specified tenant project. When set to null, users are signed in to the parent project.
    *
    * @example
    * ```javascript
-   * // Set the tenant ID on Auth instance.
+   * // Set the tenant ID on {@link Auth} instance.
```

Comment:
Does this work in code blocks?

Reply:
I don't think so. Reverted this in 2 places in this file.